encord/orm/label_row.py

```diff
@@ -289,6 +290,9 @@ class LabelRowMetadata(Formatter):
     """Only available for certain read requests"""
     is_valid: bool = True
 
+    """Only available for data groups"""
+    spaces: Optional[dict[str, SpaceInfo]] = None
```

Comment:
Make it always available for everything, by default it will have a space which refers to the only space available.